### PR TITLE
Wanleg layout updates

### PR DESCRIPTION
**Pull request by wanleg**
- add iris layout

- rename some stuff due to 40percentclub tidy-up

- wanleg userspace tweaks

### CODE REVIEW

File: keyboards/iris/keymaps/wanleg/keymap.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
+
+#include QMK_KEYBOARD_H
+#include "wanleg.h"
+
+
+#define LAYOUT_iris_base( \
```
Comment (drashna) [inline keyboards/iris/keymaps/wanleg/keymap.c]: :D

File: layouts/community/ortho_4x12/wanleg/config.h [hunk 2]
```diff
@@ -12,15 +21,15 @@
 #endif
 
 // set top left key as bootloader mode escape key on 4x4 48key layout
-#if defined(KEYBOARD_4x4) && !defined(PRO_MICRO)
+#if defined(KEYBOARD_40percentclub_4x4) && !defined(PRO_MICRO)
```
Comment (drashna) [inline layouts/community/ortho_4x12/wanleg/config.h]: Actually, you could just use: 
```suggestion
#if defined(BOOTLOADER_QMK_DFU)
```

If you check the `bootloader.mk` file, that's what it defines for the QMK DFU.

**Comment by drashna** [comment]
Looks good! Thanks!

File: keyboards/iris/keymaps/wanleg/keymap.c (hunk 1 above)
Comment (wanleg) [reply keyboards/iris/keymaps/wanleg/keymap.c]: what can i say? you're the best person to steal code from!

File: layouts/community/ortho_4x12/wanleg/config.h (hunk 2 above)
Comment (wanleg) [reply layouts/community/ortho_4x12/wanleg/config.h]: makes sense. i'll change it in my next PR

**Pull request merged**